daemon/kill.go

```diff
@@ -171,7 +171,7 @@ func (daemon *Daemon) Kill(container *containerpkg.Container) error {
 		return nil
 	}
 
-	log.G(ctx).WithError(status.Err()).WithField("container", container.ID).Errorf("Container failed to exit within %v of kill - trying direct SIGKILL", waitTimeout)
+	log.G(ctx).WithFields(log.Fields{"error": status.Err(), "container": container.ID}).Infof("Container failed to exit within %v of kill - trying direct SIGKILL", waitTimeout)
```

Comment:
Let me know what you think; we _could_ consider logging as warning as well (`Warnf`)

Reply:
If process wasn't killed with `SIGKILL` then something is probably wrong? As this is the "not nice" way of ending the process (as opposed to `SIGTERM`).

Feels to me like it should at least be a warning.

Reply:
Yeah, perhaps a `Warnf` makes sense; I'll update